lib/routes/joshwcomeau/latest.ts

```diff
@@ -0,0 +1,59 @@
+import { Data, Route } from '@/types';
+import { getRelativeUrlList, processList, rootUrl } from './utils';
+
+export const route: Route = {
+    path: '/latest/:category?',
+    categories: ['programming'],
+    example: '/latest/css',
```

Comment:
```suggestion
    example: '/joshwcomeau/latest/css',
```
example should have the namespace prepended.